recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/defconfig

```diff
@@ -588,7 +589,7 @@ CONFIG_ARM_CPUIDLE=y
 #
 # CPU Frequency scaling
 #
-# CONFIG_CPU_FREQ is not set
+CONFIG_CPU_FREQ=y
```

Comment:
Are you sure CONFIG_CPU_FREQ is needed here?

Reply:
I just wanted to be as close as possible to defconfig the bare yocto provided. I meant to make only necessary changes.

Reply:
IMHO, CONFIG_CPU_FREQ is not a good candidate to keep under XEN

Reply:
I know, but any optimizations will come in latter.

Reply:
I would rather treat it as sanitization.